### PR TITLE
[Docs] add zh-cn doc 0_config_.md

### PR DESCRIPTION
**Pull request by SheffieldCao**
## Motivation 
Fix a typo in en_docs 0_config.md

### CODE REVIEW

**Comment by SheffieldCao** [comment]
Add zh-cn document of `docs/zh-cn/tutorials/0_config_.md` . (Resolved issue #125)

File: docs/zh_cn/tutorials/0_config_.md [hunk 1]
```diff
@@ -1,3 +1,263 @@
...
+
+## 配置文件结构
+
+为了帮助用户对完整的配置和 MMFlow 中的模块有一个基本的了解，我们以在 `flyingchairs` 上使用 `slong` 训练的 PWC-Net 的配置为例进行简单的讲解。有关每个模块的更详细的用法和相应的替代方案，请参阅 API 文档和 [MMDetection 教程](https://github.com/open-mmlab/mmdetection/blob/master/docs/tutorials/config.md)。
+
+```python
+_base_ = [
+    '../_base_/models/pwcnet.py', '../_base_/datasets/flyingchairs_384x448.py',
+    '../_base_/schedules/schedule_s_long.py', '../_base_/default_runtime.py'
+]# base config file which we build new config file on.
```
Comment (MeowZheng) [inline docs/zh_cn/tutorials/0_config_.md:41]: might translate comments in code in Chinese

**Review by MeowZheng — COMMENTED** [review]
Many thanks for your contribution !

File: docs/zh_cn/tutorials/0_config_.md (hunk 1 above)
Comment (SheffieldCao) [reply docs/zh_cn/tutorials/0_config_.md]: Ok.

**Review by MeowZheng — APPROVED** [review]

**Pull request merged**